pkg/resourcequotas/list.go

```diff
@@ -0,0 +1,58 @@
+package resourcequotas
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// List returns resource quota inventory in the given namespace.
+func List(apiClient *clients.Settings, nsname string, options ...metav1.ListOptions) ([]*Builder, error) {
```

Comment:
There's no corresponding unit test for list

Reply:
Done.